aip/client-libraries/4223.md

```diff
@@ -0,0 +1,38 @@
+---
+id: 4223
+state: draft # TODO(noahdietz): Update pre-merge
+created: 2022-05-11 # TODO(noahdietz): Update pre-merge
+---
+
+# Client-side paylod validation
+
+Services document the behavior of specific request fields using the
+`google.api.field_behavior` annotation defined in [AIP-203][]. This annotation
+conveys how a service interprets the field, including how a field may be
+validated (or not) by the service.
+
+## Guidance
+
+Client libraries **must not** implement client-side payload validation based
```

Comment:
This sounds too strict, given the "Local failures" section below.  Should we add "...other then for preventing local failures, as described below" or something like that?